.github/workflows/ct-release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
-    container: registry.k1.zportal.co.uk/practically-oss/conventional-tools:1.x
+    container: practically/conventional-tools:1.x
```

Comment:
Do you think we should be using a pinned docker images? I don't want to get
into a situation where we push a bad build and are unable to release a new
version.